bundler/lib/bundler/compact_index_client.rb

```diff
@@ -5,7 +5,13 @@
 
 module Bundler
   class CompactIndexClient
+    # NOTE: MD5 is here not because we expect a server to respond with it, but
```

Comment:
Should this be TODO instead?

Reply:
I'm not sure. I mentioned this temporary md5 thing in a bunch of places so maybe we'll remember to get it.

Reply:
I mean I do rarely scan codebase for `NOTE`, but I do often for `TODO`. I'll try to keep this in my mind as well for now as a candidate to update at some point.

Reply:
Ah, I don't usually leave TODO in PRs, but that's a good point if you actually search for it. We could update.